src/sdk/pynni/nni/evolution_tuner/evolution_tuner.py

```diff
@@ -140,7 +175,12 @@ def generate_parameters(self, parameter_id, **kwargs):
             A group of candaidte parameters that evolution tuner generated.
         """
         if not self.population:
-            raise RuntimeError('The population is empty')
+            is_rand = dict()
+            for item in self.space:
+                is_rand[item] = True
+
+            config = json2parameter(self.searchspace_json, is_rand, self.random_state)
```

Comment:
Is it possible to generate duplicate parameters?

Reply:
Yes, the first generation is random generated.